packages/synapse-interface/pages/pool/poolManagement/Deposit.tsx

```diff
@@ -190,10 +207,7 @@ const Deposit = ({
       },
     }
 
-    // if (sumBigNumbersFromState().eq(0)) {
-    //   console.log(`'am hi here `)
-    //   properties.disabled = true
-    // }
+    // if (depositQuote.priceImpact)
```

Comment:
nit:

Reply:
sorry saw this a second late, addressing in my next PR for updating chain ordering list: 

https://github.com/synapsecns/sanguine/commit/65b09a3302cbb9d34ceb6e662551584909954ab7